### PR TITLE
Prettier lint

### PR DESCRIPTION
**Pull request by ascala7**

### CODE REVIEW

**Comment by ascala7** [comment]
https://manurana.medium.com/linting-formatting-in-react-and-react-native-92d7d19cc9eb

used this guide and then tried other stuff

**Comment by nathan7432** [comment]
Do we still want to do single quotes and no semi-colon?

File: src/screens/MainScreen.js [hunk 1]
```diff
@@ -1,101 +1,110 @@
...
+import { FontAwesome5, EvilIcons } from "@expo/vector-icons";
+import FadeInAnimatedText from "../components/FadeInAnimatedText";
+import RevealMessage from "../components/RevealMessage";
+import Header from "../components/Header";
+import { globalStyle, colors } from "../styles/styles";
 
 /** Screen where messages are seen. */
-const MainScreen = ( {navigation} ) => {
-  const [lastMessageTime, setLastMessageTime] = useState()
-  const { user } = useAuthentication();
```
Comment (ascala7) [inline src/screens/MainScreen.js:15]: not sure if this an issue

Comment (nathan7432) [reply src/screens/MainScreen.js:15]: if we dont need user in that screen then we don't need line 15

Comment (ascala7) [reply src/screens/MainScreen.js:15]: user was not being used

Comment (nathan7432) [reply src/screens/MainScreen.js:15]: but I was assuming we would need user on all screens at some point

File: metro.config.js
Comment (nathan7432) [inline metro.config.js:1]: This still has double quotes and semicolons is that what we want

File: src/screens/FriendsScreen.js [hunk 2]
```diff
@@ -1,60 +1,92 @@
...
       <View style={globalStyle.container}>
-        <Header showBack={true} navigation={navigation} showProfile={true}/>
+        <Header showBack={true} navigation={navigation} showProfile={true} />
         <TextInput
           style={styles.searchBar}
           placeholder="Search friends"
           value={searchQuery}
           onChangeText={handleSearch}
-          placeholderTextColor='rgba(255,255,255,.8)'
+          placeholderTextColor="rgba(255,255,255,.8)"
```
Comment (nathan7432) [inline src/screens/FriendsScreen.js]: Why double quotes here

File: src/screens/ProfileScreen.js [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
+import { signOut } from 'firebase/auth'
+import useAuthentication from '../utils/useAuthentication'
+import Header from '../components/Header'
+import { globalStyle, colors } from '../styles/styles'
 
 /** Screen where the profile can be viewed. */
-const ProfileScreen = ( {navigation} ) => {
-  const { user } = useAuthentication();
+const ProfileScreen = ({ navigation }) => {
+  useAuthentication()
```
Comment (nathan7432) [inline src/screens/ProfileScreen.js:17]: We will definitely need user here

File: src/screens/WelcomeScreen.js [hunk 4]
```diff
@@ -1,49 +1,58 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
-import FadeInAnimatedText from '../components/FadeInAnimatedText';
-import { globalStyle } from '../styles/styles';
+import { StatusBar } from "expo-status-bar";
```
Comment (nathan7432) [inline src/screens/WelcomeScreen.js]: Why double quotes here

File: src/screens/ProfileScreen.js (hunk 3 above)
Comment (ascala7) [reply src/screens/ProfileScreen.js:17]: lint will yell at you here so when we need it we can put it back

File: src/screens/WelcomeScreen.js (hunk 4 above)
Comment (ascala7) [reply src/screens/WelcomeScreen.js]: fixed

File: metro.config.js
Comment (ascala7) [reply metro.config.js:1]: file is ignored from lint

**Review by nathan7432 — APPROVED** [review]
Looks good! Great job Tony I know you worked really hard on this!

**Pull request merged**